financial-connections/src/main/java/com/stripe/android/financialconnections/features/success/SuccessContent.kt

```diff
@@ -241,7 +242,7 @@ private fun SuccessFooter(
             .testTag("done_button")
             .fillMaxWidth()
     ) {
-        Text(
+        MiddleEllipsisText(
```

Comment:
Noticed on large merchants the button looks ugly on the success screen! This fixes it (see video)